summary_workflows/quantification/corr_with_gt.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import pearsonr
+import json
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('bed', help='The BED file containing predictions merged with ground truth.')
+args = parser.parse_args()
+
+fname = args.bed
+out = pd.read_csv(fname, delimiter='\t', header=None)
```

Comment:
Here you could read in the csv with specific column names. As this is known and fixed, I don't see a reason to hardcode this.